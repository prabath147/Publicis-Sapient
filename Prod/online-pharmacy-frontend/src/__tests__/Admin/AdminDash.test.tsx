import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import AdminDash from '../../components/pages/Admin/adminDash/adminDash';
import mockAxios from '../../__mocks__/axios';

it('Testing Manager Dashboard component', () => {
    mockAxios.get.mockImplementationOnce(() => Promise.resolve({data: 10}));
    mockAxios.get.mockImplementationOnce(() => Promise.resolve({data: {totalRecords:100}}));
    mockAxios.get.mockImplementationOnce(() => Promise.resolve({data: {totalRecords:200}}));
    mockAxios.get.mockImplementationOnce(() => Promise.resolve({data: 20}));
    mockAxios.get.mockImplementationOnce(() => Promise.resolve({data: 30}));

    render(
        <BrowserRouter>
            <AdminDash />
        </BrowserRouter>
    );

    
    expect(screen).toMatchSnapshot();
});

