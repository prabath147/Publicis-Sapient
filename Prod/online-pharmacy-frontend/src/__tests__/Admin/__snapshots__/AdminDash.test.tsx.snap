// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Manager Dashboard component 1`] = `Screen {}`;
